.github/ISSUE_TEMPLATE/access-issues.md

```diff
@@ -0,0 +1,166 @@
+---
+name: Network Access Issues
+about: Let us know if you're having trouble reaching PyPI 
+
+---
+
+<!--
+    NOTE: This issue should be for problems accessing PyPI itself, including:
+    * pypi.org
+    * test.pypi.org
+    * files.pythonhosted.org
+
+    This issue should NOT be for any non-PyPI properties (like
+    python.org, docs.python.org, etc.)
+-->
+
+## My Platform
+<!--
+    Any details about your specific platform:
+    * If the problem is in the browser, what browser, version, and OS?
+    * If the problem is with a command-line tool, what version of that tool?
+    * If the problem is with connecting to PyPI, include some details about
+      your network, including SSL/TLS implementation in use, internet service
+      provider, and if there are any firewalls or proxies in use.
+-->
+
+## Network telemetry
+<!--
+    Providing this information is crucial in helping diagnose networking
+    issues for PyPI.
+-->
+
+<!--
+### Fastly Debug
+    Note that this output is optional, as it encodes identifying information
+    about your browser, public IP, and location.
+
+    We'll remove this output from the issue before closing, but do not feel
+    comfortable making it requried for all users.
+
+    If you have a browser that reproduces your access issue, go to
+    https://fastly-debug.com, when the page has loaded copy and paste the
+    base64 encoded blob at the top of the page into the box below, and
+    uncomment this section.
+
+```
+<Replace with your output>
+```
+-->
+
+### DNS Resolution
+
+<!--
+    Provide the servers you're querying for DNS here if you know. They can be
+    found in /etc/resolv.conf on most linux servers, or your network settings
+    for MacOS and Windows.
+-->
+
+```shell
+dig pypi.org A
```

Comment:
Is it worth mentioning what dig is and how to get it (like is done for traceroute below), or is that pretty standard on modern *ix-like OSs? (It is installed by default on macOS 10.13 as is `host`.)